hate-speech-api/hate_speech_clf_api.py

```diff
@@ -0,0 +1,80 @@
+import sys
+import pickle
+from flask import Flask, jsonify, abort, request
+from flask_classful import FlaskView, route
+import data_processing.mm_segmenter as mm
+from data_processing.mm_converter import zawgyi_to_unicode
+import re
+import json
+from sklearn.ensemble import RandomForestClassifier
+
+app = Flask(__name__)
+
+class HateSpeechClassifier(FlaskView):
+
+    def __init__(self):
+        self.model = "None"
+        self.burmese_segmenter = mm.Segmenter()
+        self.clf_mapping = {
+            "RF" : "clfs/oversampled-RFClf.pkl"
+        }
+        with open("clfs/oversampled-tfidf-vect.pkl", 'rb') as vect_b:
```

Comment:
why do we have the tdidf model?

Reply:
I added ngrams vectorizer and models under model/.  But ngrams doesn't perform as well as tfidf.